core/templates/dev/head/services/ExplorationContextService.js

```diff
@@ -110,6 +110,11 @@ oppia.factory('ExplorationContextService', [
       isInExplorationEditorMode: function() {
         return (this.getPageContext() === PAGE_CONTEXT.EDITOR &&
             this.getEditorTabContext() === EDITOR_TAB_CONTEXT.EDITOR);
+      },
+
+      // To be used for testing purposes only.
```

Comment:
Is this okay, or is there a better approach here?

Reply:
This isn't OK. But in a test you can mock functions in the explorationContextService. You could use `spyOn(explorationContextService, 'getExplorationId').and.returnValue(...);`, for example.

Reply:
Oh, thanks! Done